src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableEntityContext.cs

```diff
@@ -169,5 +171,25 @@ public interface IDurableEntityContext
         /// </exception>
         /// <returns>The instance id of the new orchestration.</returns>
         string StartNewOrchestration(string functionName, object input, string instanceId = null);
+
+        /// <summary>
+        /// Dynamically dispatches the incoming entity operation using reflection.
```

Comment:
Just for my own understanding, we _have_ to use reflection because the entity is not known statically, right?

Reply:
Not sure what you mean by this. 

Essentially we use reflection to create the object given the type they provide, because that is the toolset that C# gives us to create objects given a specific type.

Reply:
That answers my question :) 
To answer yours, what I meant by "not known statically" is simply that `T` can only be known at runtime, which you're saying is true!